react-prebuild.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react';
+import {renderToString} from 'react-dom/server';
+import Logo from './kuma/javascript/src/Logo';
+
+let stdinAccumulator = '';
+
+process.stdin.on('data', input => {
+    stdinAccumulator += input.toString();
+});
+
+process.stdin.on('end', () => {
+    const locales = JSON.parse(stdinAccumulator);
+
+    const result = Object.keys(locales).reduce((acc, locale) => ({
+        [locale]: {
+            Logo: renderToString(<Logo
+                url={locales[locale].Logo.props.url}
+            />),
+        },
+        ...acc
+    }), {});
+
+    console.log(JSON.stringify(result, null, 2))
```

Comment:
I'm guessing that it would be useful to have a try/catch block above and print errors to stderr so that they can be reported by the management command. And maybe set th eprocess exit code when an error occurs? (The point is that I'd want to be sure that the build process fails if SSR fails.)